package.json

```diff
@@ -37,7 +42,7 @@
     },
     "clientPaths": {
       "devkit": "src/clientapi",
-      "ff": "src/clientapi/api/ff"
+      "*": "src/clientapi/api/"
```

Comment:
This will require at least a `minor` release since this exposes new built in import paths.